File: mediscreen-ui/src/app/models/patient.ts
export class Patient {

    public id: number;
    public firstName: string;
    public lastName: string;
    public sex: string;
    public address: string;
    public city: string;
    public phone: string;
    public dateOfBirth: Date;
    public note: string | undefined;

    constructor(id: number, firstName: string, lastName: string,
                sex: string, address: string, city: string, phone: string, dateOfBirth: Date) {
      this.id = id;
      this.firstName = firstName;
      this.lastName = lastName;
      this.sex = sex;
      this.address = address;
      this.city = city;
      this.phone = phone;
      this.dateOfBirth = dateOfBirth;
    }


}

File: mediscreen-diabetesia/src/main/java/com/mediscreen/mediscreendiabetesia/proxy/Patient.java
package com.mediscreen.mediscreendiabetesia.proxy;

import java.time.LocalDate;

import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateDeserializer;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
public class Patient {
	
	 private Integer id;
	 private String firstName;
	 private String lastName;
	 
	 @JsonDeserialize(using = LocalDateDeserializer.class)
	 private LocalDate dateOfBirth;
	 private String sex;
	 private String phone;
	 private String address;
	 private String city;
	 
	public Patient(Integer id, LocalDate dateOfBirth, String sex) {
		this.id = id;
		this.dateOfBirth = dateOfBirth;
		this.sex = sex;
	}
	
	public Patient(String firstName,
			String lastName,LocalDate dateOfBirth,
			String sex,
			String phone,
			 String address,
			String city) {
		
		this.firstName = firstName;
		this.lastName = lastName;
		this.dateOfBirth = dateOfBirth;
		this.sex = sex;
		this.phone = phone;
		this.address = address;
		this.city = city;
	}
	 
	 
}
